packages/webapp/pages/history.tsx

```diff
@@ -27,6 +40,8 @@ const History = (): ReactElement => {
   const router = useRouter();
   const tabQuery = router.query?.t?.toString() as HistoryType;
   const [page, setPage] = useState(HistoryType.Reading);
+  const { isNewMobileLayout } = useMobileUxExperiment();
```

Comment:
Since there are so many checks in this file and they render different components, would it be worth to have 2 different components with mobile layout and without mobile layout?

Reply:
I don't mind it as it is for now, in the interest of time

Reply:
let's see then if it's worth doing once this wins as an experiment (actually it's not going to be an experiment, just a feature flag...) I think we're confusing things with the hook name

Reply:
Almost think indeed this should happen on level up.
feels a bit like abusing this page now, but not blocking.

Reply:
isn't this already at the top level? -- also, since all these `isNewMobileLayout` checks are temporary things until we fully enable this change, this `abuse` will not be permanent 😅

Reply:
The layout would be more top level :)